src/checkpoint/checkpoint_txn.c

```diff
@@ -1441,6 +1441,11 @@ __checkpoint_db_internal(WT_SESSION_IMPL *session, const char *cfg[])
     if (ckpt_crash_before_metadata_sync)
         __wt_debug_crash(session);
 
+    WT_STAT_CONN_SET(session, checkpoint_stop_stress_active, 1);
```

Comment:
Is there an explanation why this was moved?

Reply:
This was moved to allow it to be repurposed for hitting this exact point we need. The only other test it's used for is `test_rollback_to_stable35` and it still serves that purpose. I can add a comment on this stress point?

Reply:
I think it would be great to add a comment explaining why this stress point is here, indeed.

Reply:
Updated to 
>      * Stress point to stop just before we sync the metadata file. Used to recreate log recovery scenarios with an incomplete checkpoint.

I've worded to be very general.